src/javaharness/java/arcs/api/DeviceClientImpl.java

```diff
@@ -1,6 +1,7 @@
 package arcs.api;
 
 import java.util.logging.Logger;
+
```

Comment:
nit, please, revert this file, no changes here.

Reply:
Done.